draft-ietf-rats-reference-interaction-models.md

```diff
@@ -326,6 +326,74 @@ As soon as the Verifier receives the Evidence and the Event Logs, it appraises t
 Appraisal procedures are application-specific and can be conducted via comparison of the Claims with corresponding Reference Values, such as Reference Integrity Measurements.
 The final output of the Verifier are Attestation Results. Attestation Results constitute new Claim Sets about the properties and characteristics of an Attester, which enables Relying Parties, for example, to assess an Attester's trustworthiness.
 
+### Models and example sequences of Challenge/Response Remote Attestation
+According to the RATS Architecture, two reference models for Challenge/Response Attestation have been proposed. This sections highlights the information flows bewteen the Attestor, Verifier and Relying Party undergoing Remote Attestation Procedure, using these models.
+
+1. Passport Model
+
+The passport modle is so named because of its resemblance to how nations issue passports to their citizens. In this Model, the attestation sequence is a
```

Comment:
```suggestion
The passport model is so named because of its resemblance to how nations issue passports to their citizens. In this model, the attestation sequence is a
```